app/scripts/modules/core/src/pipeline/config/stages/script/scriptExecutionDetails.controller.ts

```diff
@@ -0,0 +1,32 @@
+import { module } from 'angular';
+import { get } from 'lodash';
+import { StateParams } from '@uirouter/angularjs';
```

Comment:
prefer `@uirouter/core` when possible.

(note: not your fault; `@uirouter/angularjs` does `export * from `@uirouter/core` so it works either way)